### ISSUE TITLE
Document how to use Mapnik 2.2 with Kosmtik

### ISSUE THREAD

**Issue by pnorman** [kosmtik/kosmtik#72]
Many stylesheets are not yet compatible with Mapnik 3. It would be worth having instructions on how to run Kosmtik with Mapnik 2.2.

**Comment by yohanboniface** [comment]
You mean changing the version in package.json or changing mapnik-reference from command line (--mapnik-version) ?

**Comment by pnorman** [comment]
The version of Mapnik used for rendering

**Comment by yohanboniface** [comment]
Actually, I'm not sure that's easy to do :/
AFAIK, passing `--mapnik-version` changes the mapnik-reference version carto uses for validating the carto project, but I don't think this actually also changes the version of the *Mapnik* engine.
Kosmtik only gives this parameter to carto btw, not to Mapnik, IIRC.
BTW, seems the parameter is badly named (and I'm guilty ;) ).
So if you want to have mapnik 2.2 running, I guess you need to install it instead. Thus changing the dependency in `package.json` or installing by hand with `npm install mapnik@x.y.z`. But I don't know from which version node-mapnik installs itself mapnik, so this may ends up with a complex situation. Not to mention some conflicts in dependencies that could arise going this way :/

In summary, I'm afraid the only thing that may be documented (more than being on `--help`) is the `--mapnik-version` parameter, but I'm afraid it's only used for validating the carto project and generating the Mapnik XML, not to magically changes the Mapnik engine that will create the final tiles.

ping @springmeyer, I may be missing a point.

**Comment by nebulon42** [comment]
I think this is out of scope for us out of the reasons @yohanboniface mentioned and Mapnik 2 gets less and less important anyway. Closing.
